src/client/components/responsive-image/responsive-image.vue

```diff
@@ -104,5 +104,7 @@ export default {
 .responsive-image__caption {
   margin-top: var(--spacing-half);
   text-align: center;
+  color: var(--text-light-color);
```

Comment:
Videos also have a caption. I would say move `color`, `font-style` and `text-align` to core styling `figcaption`.